### PR TITLE
Review and booking controllers

### PR DESCRIPTION
**Pull request by Sargeant91**
@janm97 
@JohnMarsden24 

I've added the routes for bookings and reviews, I've also added the skeleton controllers.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   devise_for :users
   resources :users do
     resources :bookings, except: [:index, :show, :destroy, :edit, :update]
```
Comment (janm97) [inline config/routes.rb]: Wouldn't it be more readable to use `only: [:create, :new]` ?

File: config/routes.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
-    resources :reviews, except: [:index, :show, :destroy]
+    resources :reviews, except: [:index, :show, :destroy, :edit, :update]
```
Comment (janm97) [inline config/routes.rb]: I think `only: [:create]` should do it. Don't think we need `:new` as we are planning on displaying the 'new comment' form at the bottom of the artists page?

File: app/controllers/reviews_controller.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class ReviewsController < ApplicationController
+
+  def create
+  end
+
+  def edit
+  end
+
+  def update
+  end
```
Comment (janm97) [inline app/controllers/reviews_controller.rb]: We aren't creating resources in routes for this. Don't think we'd wan't users to update reviews anyways though?

Comment (Sargeant91) [reply app/controllers/reviews_controller.rb]: I think if you're the user that left the review you should be able to edit and delete, we can just do some validation in those methods?

**Pull request merged**